### PR TITLE
Omit hermit action in merge code

### PR DESCRIPTION
**Pull request by moukoublen**
### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

#### Introducing a new rule?

- [ ] Generate rule metadata using [this script](https://github.com/elastic/cloudbeat/tree/main/security-policies/dev#generate-rules-metadata)
- [ ] Add relevant unit tests
- [ ] Generate relevant rule templates using [this script](https://github.com/elastic/cloudbeat/tree/main/security-policies/dev#generate-rule-templates), and open a PR in [elastic/packages/cloud_security_posture](https://github.com/elastic/integrations/tree/main/packages/cloud_security_posture)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -42,8 +42,12 @@ jobs:
           # If the event is push to branch use the default ref.
           # If the event is pull request (`pull_request_target` in our case) use merge commit as ref to run lint over the PR's code.
 
-      - name: Hermit Environment
-        uses: ./.github/actions/hermit
+      - name: Initialize hermit
```
Comment (orouz) [inline .github/workflows/ci.yml:45]: can you clarify please why this change is needed? i'd assume that sharing this step is preferable? (caching mostly)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ jobs:
           init-tools: 'true'
 
   lint:
-    needs: [ init-hermit ]
+    #needs: [ init-hermit ]
```
Comment (orouz) [inline .github/workflows/ci.yml:33]: maybe delete this instead of comment out?

Comment (moukoublen) [reply .github/workflows/ci.yml]: Done

**Review by orouz — APPROVED** [review]
